xmodule/modulestore/__init__.py

```diff
@@ -1393,12 +1393,3 @@ def only_xmodules(identifier, entry_points):
     from_xmodule = [entry_point for entry_point in entry_points if entry_point.dist.key == 'xmodule']
 
     return default_select(identifier, from_xmodule)
```

Comment:
@0x29a, just thinking - `only_xmodules` seems to be used only in tests, so we could move it somewhere else (e.g. `xmodules.tests.helpers`). We could also remove it if it's not needed anymore. What do you think?

If it makes sense, we can create a small follow-up for this. We don't need to extend the scope of the current ticket.